### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by chiragkhimani**
I completed login feature. Can everyone please review?

### CODE REVIEW

**Comment by chiragkhimani** [comment]
Looks good....great job Swati

**Comment by AunjunH** [comment]
looks good!
great Swati

**Comment by VladGribov** [comment]
Alfredo Sauce

**Comment by aigulcross** [comment]
Great job!

**Comment by AlexNikov24** [comment]
Good job

**Comment by Tryastsya** [comment]
MyComment)

**Review by guzalcali — COMMENTED** [review]
Nicely done!

**Comment by jamnova12** [comment]
Nice job! 💯

**Comment by AyaNokerova** [comment]
Fantastic!!

**Comment by VictoriaSharapova** [comment]
Good job

**Comment by OlegKinakh** [comment]
hi

**Comment by TarikIbi** [comment]
Great !

**Comment by SwatiPanda** [comment]
Thanks Chirag!

**Comment by gitKnowsMe** [comment]
Good job

**Comment by M1keFreeman** [comment]
Nice Job

**Comment by sagyn11** [comment]
well done

**Comment by blgra** [comment]
Nice...

**Comment by Alice312** [comment]
Hi :)

File: src/com/java/class22/GitPractice.java [hunk 1]
```diff
@@ -2,9 +2,16 @@
...
     public static void main(String[] args) {
+
         System.out.println("This is from Chirag's PC");
         System.out.println("gitKnowsMe was here");
         System.out.println("TestingMyFirstProgram");
+
+        System.out.println("TestingHomeFeature-Swati");
+        System.out.println("Testing");
+        System.out.println("Login Feature Started");
+        System.out.println("Login feature completed till teh time manager is coming!! yay!");
```
Comment (Tryastsya) [inline src/com/java/class22/GitPractice.java:13]: Yay!)

Comment (chiragkhimani) [inline src/com/java/class22/GitPractice.java:13]: what do you mean by the manager is coming????? it doesn't looks good so remove it.

File: src/com/java/class22/GitPractice.java [hunk 2]
```diff
@@ -6,9 +6,12 @@ public static void main(String[] args) {
         System.out.println("This is from Chirag's PC");
         System.out.println("gitKnowsMe was here");
         System.out.println("TestingMyFirstProgram");
+
         System.out.println("TestingHomeFeature-Swati");
         System.out.println("Testing");
         System.out.println("Login Feature Started");
         System.out.println("Login feature completed till teh time manager is coming!! yay!");
```
Comment (jamnova12) [inline src/com/java/class22/GitPractice.java:13]: 💯

**Comment by LizWilcox217** [comment]
Good job with login.

File: src/com/java/class22/GitPractice.java (hunk 1 above)
Comment (guzalcali) [inline src/com/java/class22/GitPractice.java:13]: Not bad

**Comment by SeviliaM** [comment]
Perfect

**Comment by Anzhelav** [comment]
Cool!

**Pull request merged**